web/Services/PropertyService.cs

```diff
@@ -191,16 +191,31 @@ public List<PropertyDbModel> SortProperties(string? by)
         return properties;
     }
 
-    public PropertyCountModel CountProperties()
+    public PropertyCountModel CountProperties(int? landlordId = null)
     {
-        var propertyCounts = new PropertyCountModel
+        if(landlordId == null)
         {
-            RegisteredProperties = _dbContext.Properties.Count(),
+            return new PropertyCountModel
+            {
+                RegisteredProperties = _dbContext.Properties.Count(),
+                LiveProperties = _dbContext.Properties.Count(p =>
+                    p.Availability != AvailabilityState.Draft 
+                    && p.Landlord.CharterApproved),
+                AvailableProperties = _dbContext.Properties.Count(p => p.Availability == AvailabilityState.Available)
+            };   
+        }
+
+        return new PropertyCountModel
+        {
+            RegisteredProperties = _dbContext.Properties.Count(p => p.LandlordId == landlordId),
             LiveProperties = _dbContext.Properties.Count(p =>
-                p.Availability != AvailabilityState.Draft && p.Landlord.CharterApproved),
-            AvailableProperties = _dbContext.Properties.Count(p => p.Availability == AvailabilityState.Available)
+                p.Availability != AvailabilityState.Draft 
+                && p.Landlord.CharterApproved 
+                && p.LandlordId == landlordId),
+            AvailableProperties = _dbContext.Properties.Count(p =>
+                p.Availability == AvailabilityState.Available 
+                && p.LandlordId == landlordId)
         };

```

Comment:
This will do at least 3 db lookups, instead we could just get the landlord and include the properties like this
```

        var landlord = _dbContext.Landlords.Include(l => l.Properties).SingleOrDefault(l => l.Id == landlordId);
        var properties = landlord?.Properties ?? new List<PropertyDbModel>();

        return new PropertyCountModel
        {
            RegisteredProperties = properties.Count,
            LiveProperties = properties.Count(p =>
                landlord?.CharterApproved == true && p.Availability != AvailabilityState.Draft),
            AvailableProperties = properties.Count(p =>
                p.Availability == AvailabilityState.Available)
        };